components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/ConfigurationFacade.java

```diff
@@ -171,62 +183,46 @@ public ExternalIdPConfig getIdPConfigByRealm(String realm, String tenantDomain)
     }
 
     public String getAuthenticationEndpointURL() {
-        String authenticationEndpointURL = FileBasedConfigurationBuilder.getInstance().getAuthenticationEndpointURL();
-        if (StringUtils.isBlank(authenticationEndpointURL)) {
-            authenticationEndpointURL = "/authenticationendpoint/login.do";
```

Comment:
In the old implementation, we simply return the default context if we cannot find the corresponding value in the file. 
I highly doubt whether this would work, since it will try to do a relative redirect from the callee.

For example: if I initiated an oauth2 request, the redirect happens to
https://localhost:9443/oauth2/authenticationendpoint/login.do

@malithie is this something we need to fix?